vertica_python/tests/integration_tests/test_cursor.py

```diff
@@ -401,6 +402,30 @@ def test_format_quote_unicode(self):
             bad_word = u'Fr\xfchst\xfcck'
             formatted_word = u''.join((u'"', re.escape(bad_word), u'"'))
             self.assertEqual(formatted_word, cur.format_quote(bad_word, True))
+            
+    def test_datetime_types(self):
+        with self._connect() as conn:
+            cur = conn.cursor()
+
+            # clean old table
+            cur.execute("DROP TABLE IF EXISTS {0}".format(self._table))
+
+            # create test table
+            cur.execute("""CREATE TABLE {0} (
+                                a INT,
+                                b VARCHAR(32),
+                                c TIMESTAMP,
+                                d DATE
+                           )
+                        """.format(self._table))
+
+            cur.execute("INSERT INTO {0} (a, b, c, d) VALUES (:n, :s, :t, :d)".format(self._table),
+                            {'n': 10, 's': 'aa', 't': datetime.datetime(2018, 9, 7, 15, 38, 19, 769000), 'd': datetime.date(2018, 9, 7)})
```

Comment:
The test looks good. Feel free to reuse `self._table` or create your new table, as long as you clean up tables at the end.

Last thing, don't forget to add a `TIME` column. Then you are good to go :)